crunchydb/high-availablility/PostgresCluster.yaml

```diff
@@ -5,13 +5,39 @@ metadata:
 spec:
   monitoring:
     pgmonitor:
+      # this stuff is for the "exporter" container in the "hippo-ha-pgha1" set of pods
       exporter:
         image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.4-0
```

Comment:
Any `image` tag should be removed and left to the operator's default. Setting an image may lead to incompatibilities when updating the operator